strimzi/tests/README.md

```diff
@@ -0,0 +1,49 @@
+# Overview
+
+The e2e environment spins up a kafka cluster with a zookeeper and a simple hello world connector.
+
+# Useful command line
+
+Run the command inside the kind container: 
+
+```commandline
```

Comment:
Optional: shell leaves a $ symbol to indicate the command line
```suggestion
```shell
```

Reply:
Didn't know about that, thanks!